packages/tabs/src/index.js

```diff
@@ -192,12 +218,12 @@ export const Tab = forwardRef(function Tab(
       data-reach-tab=""
       ref={ref}
       role="tab"
-      id={makeId("tab", _id)}
+      id={makeId("tab", makeId(_id, index))}
       tabIndex={isSelected ? 0 : -1}
       aria-selected={isSelected}
-      aria-controls={makeId("panel", _id)}
+      aria-controls={makeId("panel", makeId(_id, index))}
```

Comment:
The composed call to `makeId` is used here because we have [removed](https://github.com/reach/reach-ui/pull/361/files#diff-6c8cbe1adb417474a2ffacc00dbd2c22L107) it happening from `<TabList>` and `<TabPanels>` when it was cloned to their children. Happy to move it to a variable or something to make it easier to look at if desired.